src/main/helpers/controller/ClaimantContactDetailsHelper.ts

```diff
@@ -0,0 +1,64 @@
+import { AppRequest } from '../../definitions/appRequest';
+import { CaseWithId } from '../../definitions/case';
+import { TranslationKeys } from '../../definitions/constants';
+import { SummaryListRow, addSummaryRow } from '../../definitions/govuk/govukSummaryList';
+import { AnyRecord } from '../../definitions/util-types';
+import { answersAddressFormatter } from '../AddressHelper';
+
+import { isClaimantRepresentedWithMyHMCTSCase } from './ContactTribunalHelper';
+
+export const getClaimantContactDetails = (req: AppRequest): SummaryListRow[] => {
+  const userCase = req.session?.userCase;
+  const translations: AnyRecord = {
+    ...req.t(TranslationKeys.CLAIMANT_CONTACT_DETAILS, { returnObjects: true }),
+  };
+  if (isClaimantRepresentedWithMyHMCTSCase(userCase)) {
```

Comment:
This only caters for the C's Rep being on the portal but it is also possible for them to be offline. This should probably be changed such that it should be consider the rep regardless if they have portal access or not

Reply:
A new method for the if as opposed to changing the isClaimantRepresentedWithMyHMCTSCase method